alfalfa_web/package.json

```diff
@@ -10,7 +10,7 @@
     "clean": "npx rimraf build",
     "start": "node ./build/index.js",
     "start-dev": "npm run build-server && env-cmd -f ../.env npm start",
-    "watch": "webpack --watch --progress"
+    "watch": "nodemon -e js,html --ignore build/ --exec \"npm run build-app && npm run build-server && npm start\""
```

Comment:
this is the main change to make the watch code work. I couldn't see a way to build the server and the app with just webpack. Maybe I missed something.